docs/installation/linux/ubuntulinux.md

```diff
@@ -88,7 +88,7 @@ packages from the new repository:
     > **Note**: Docker does not provide packages for all architectures. To install docker on
     > a multi-architecture system, add an `[arch=...]` clause to the entry. Refer to the
     > [Debian Multiarch wiki](https://wiki.debian.org/Multiarch/HOWTO#Setting_up_apt_sources)
-    > for details.
+    > for details.The repos don't support all arches, but you can find the binaries here https://master.dockerproject.org.
```

Comment:
`arches` is not a word :grin: 

>  Docker does not provide packages for all architectures.

That's pretty explanatory for me, tbh, but it can be helpful to add a link to the nightly builds, what about?

> You can find nightly built binaries in https://master.dockerproject.org.